modules/tinymce/tools/docs/tinymce.EditorManager.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * This is the base class for the link:../tinymce.root/[`tinymce`] global, a factory for link:../tinymce.editor/[`tinymce.Editor`] instances.
+ *
+ * The global link:../tinymce.root/[`tinymce`] is the only one expected to be created.
```

Comment:
I’m reading this as if _link:../tinymce.root[`<string>`]_ are noun or adjectival phrases: denoters of discrete, identifiable things. Sort of, but not really, treating the strings as [bleeps](https://gocomics.com/peanuts/1964/11/04).

And, the editorial consequence of this reading is thus:

```suggestion
 * This is the base class for the link:../tinymce.root/[`tinymce`] global.
*
* This class is a factory for link:../tinymce.editor/[`tinymce.Editor`] instances.
*
* And the link:../tinymce.root/[`tinymce`] global is the only
* EditorManager instance the class is expected to create.
```

Reply:
I'm not a fan of the bleeps and bloops but it's the only way to make moxiedoc + antora generate a link in the final rendering.